www/index.html

```diff
@@ -38,23 +20,154 @@
     -->
     <!-- <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: 'unsafe-inline' https://ssl.gstatic.com; style-src 'self' 'unsafe-inline'; media-src *" /> -->
 
-    <link rel="stylesheet" type="text/css" href="css/index.css" />
-    <title>Hello World</title>
+    <link rel="stylesheet" href="./libs/bootstrap/3.3.7/css/bootstrap.min.css" >
+    <link rel="stylesheet" href="./libs/font-awesome/4.6.3/css/font-awesome.min.css" >
+
+    <link rel="stylesheet" href="./css/index.css" >
+
+    <title>Fln Routes</title>
 </head>
 
 <body>
-    <div class="app">
-        <h1>PhoneGap</h1>
-        <div id="deviceready" class="blink">
-            <p class="event listening">Connecting to Device</p>
-            <p class="event received">Device is Ready</p>
+
+    <div class="container">
+        <div class="page-header">
+            <h1>Fln Routes</h1>
+            <p class="lead">Simple Florianópolis' public transportation route searcher.</p>
+        </div>
+
+        <div class="list-view">
+            <form action="#" class="form-inline route-search-form">
+                <label class="sr-only" for="location">Location</label>
+                <input type="text" id="location" name="location" class="form-control" placeholder="Location">
+                <button type="submit" class="btn btn-primary">Go! <i class="fa fa-spinner fa-pulse fa-fw loading-indicator"></i></button>
+            </form>
+
+            <div class="routes-panel"></div>
+        </div>
+
+        <div class="detail-view hide">
+            <button type="button" class="btn pull-right back-trigger">Voltar</button>
+            <div class="container-fluid">
+                <h4 class="route-name-label"></h4>
+                <i class='fa fa-spinner fa-pulse fa-2x fa-fw loading-indicator loading-indicator'></i>
+                <div class="row">
+                    <div class="col-md-6 stops-panel"></div>
+                    <div class="col-md-6 departures-panel"></div>
+                </div>
+            </div>
+            <button type="button" class="btn pull-right back-trigger">Voltar</button>
         </div>
     </div>
-    <script type="text/javascript" src="cordova.js"></script>
-    <script type="text/javascript" src="js/index.js"></script>
-    <script type="text/javascript">
-        app.initialize();
+
+    <script id="no-input-error-tmpl" type="x-tmpl-mustache">
+        <div class="alert alert-danger alert-dismissible fade in" role="alert">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+            </button>
+            <strong>Oops!</strong> É preciso definir um nome de localidade.
+        </div>
     </script>
+
+    <script id="stops-tmpl" type="x-tmpl-mustache">
```

Comment:
I chose mustache because:

1. it's syntax is very simple and quite of enforces no logic at template level
2. it's highly used and well documented
3. it's implemented in great variety of languages (ruby, js, python, java and so on) what make it easy to migrate and also most likely to be understood by developer coming from other languages...